### ISSUE TITLE
Code Blocks: Use Dark Theme cannot be reapplied

### ISSUE THREAD

**Issue by bennyn** [Requarks/wiki#196]
1. Open "Color Theme" settings
1. Disable "Use Dark Theme" in "Code Blocks" section
1. Save changes (Button "Use Dark Theme" is disabled)
1. Check a Wiki page which uses code highlighting (you can see that there is a light theme applied)
1. Switch back to "Color Theme" settings
1. You will see now that "Use Dark Theme" is activated (although it has been deactivated) but the code highlighting still uses a light theme and it's not possible to revert to the dark theme anymore

**Comment by NGPixel** [comment]
Good catch. As a workaround, simply restart wiki.js to revert to the settings in config.yml (in which dark theme is active by default)
